tools/build_defs/cc_toolchain_util.bzl

```diff
@@ -246,8 +236,9 @@ def get_env_vars(ctx):
     return vars
 
 def is_debug_mode(ctx):
-    # see workspace_definitions.bzl
-    return str(True) == ctx.attr._is_debug[config_common.FeatureFlagInfo].value
+    # Compilation mode currently defaults to fastbuild. Use that if for some reason the variable is not set
+    # https://docs.bazel.build/versions/master/command-line-reference.html#flag--compilation_mode
+    return ctx.var.get("COMPILATION_MODE", "fastbuild") == "dbg"
```

Comment:
This matches the previous default of the `_is_debug` attribute defaulting to `False`

https://github.com/bazelbuild/rules_foreign_cc/blob/00c274b7570056239ef2e1bfdda4a658c5a135a0/workspace_definitions.bzl#L18-L32

https://github.com/bazelbuild/rules_foreign_cc/blob/00c274b7570056239ef2e1bfdda4a658c5a135a0/tools/build_defs/framework.bzl#L168-L170